### PR TITLE
[WIP] add basic verification for the login page 

### LINKED ISSUE #84: Add validation and error display on the login form

### PR DESCRIPTION
**Pull request by maximest-pierre**
Right now I can make the invalid field red but there is no prompt can somebody with a little more experience with semantic-ui can help me.

fix #84

### CODE REVIEW

**Comment by manumilou** [comment]
@maximest-pierre I'll have a look.

File: django/santropolFeast/page/templates/registration/login.html [hunk 1]
```diff
@@ -36,6 +36,41 @@ <h2 class="ui teal image header">
         <button class="ui fluid large teal button" type="submit">Login</button>
     </div>
```
Comment (manumilou) [inline django/santropolFeast/page/templates/registration/login.html]: Add here: <div class="ui error message"></div>
to load error messages.

File: django/santropolFeast/page/templates/registration/login.html [hunk 2]
```diff
@@ -36,6 +36,41 @@ <h2 class="ui teal image header">
...
 </form>
+
```
Comment (manumilou) [inline django/santropolFeast/page/templates/registration/login.html]: Unnecessary, as semanticUi JS and Jquery are already loaded by the base template.
But be careful to execute the JS on document.ready.

**Comment by RignonNoel** [comment]
**First part :**
Here the JS error message work only when i submit the form. But it's hide directly when i correct the error.

We need instant error, also we can simply write a message of the context processor..

**Second part :**
I think it's important to display error on the top of the form. If we have a bad resolution we can not see the message.

**Comment by manumilou** [comment]
It is all good to me. Just the message on failed authentication that remain to be added.

**Pull request merged**